File: 3379-score-of-a-string/3379-score-of-a-string.java
class Solution {
    public int scoreOfString(String s) {
        int rank=0;
        char[] ch=s.toCharArray();
        for(int i=0;i<ch.length-1;i++)
        {
            int v=Math.abs(ch[i]-ch[i+1]);
            rank+=v;
        }
        return rank;
    }
}

File: 3379-score-of-a-string/3379-score-of-a-string.py
class Solution:
    def scoreOfString(self, s: str) -> int:
        sc=0
        for i in range(0,len(s)-1):
            r=abs(ord(s[i+1])-ord(s[i]))
            #print(r,end=' ')
            sc+=r
            #print(sc)
        return sc
        